server/src/controllers/transaction.ts

```diff
@@ -35,7 +35,7 @@ const getTotalAmount = async (req: Request, res: Response, next: NextFunction) =
 
 const getUniquePerDay = async (req: Request, res: Response, next: NextFunction) => {
   try {
-    let data = await AboutTransactionProvider.getAll();
+    let data = await AboutTransactionProvider.getAll(0,req.query.limit && parseInt(req.query.limit));
```

Comment:
add limit.ts to src/helpers:
```
const limitOffset = req => {
  let { limit } = req.query
  let result = [0]

  if (limit) {
    limit = parseInt(limit)
  }

  if (!Math.isNaN(limit) && (limit > 0)) {
    result.push(limit)
  }

  return result
}

```

Then re-use this helper pls:

`let data = await AboutTransactionProvider.getAll(...limitOffset(req));`

Reply:
done